### PR TITLE
apply clang-format to FWCore

### PR DESCRIPTION
**Pull request by davidlange6**
As with the clang-tidy changes, this is for comment only at this point [and in any case, it does not merge....]

clang-format --style=Google -i `find FWCore -name \*.cc` `find FWCore -name \*.h`

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by Dr15Jones** [comment]
Not a fan of the no indention for namespace blocks. I guess they did that since in their code everything is probably in a namespace.

**Comment by Dr15Jones** [comment]
What is the maximum line length being used?

File: FWCore/Common/interface/LuminosityBlockBase.h [hunk 1]
```diff
@@ -6,9 +6,11 @@
 // Package:     FWCore/Common
 // Class  :     LuminosityBlockBase
 //
-/**\class LuminosityBlockBase LuminosityBlockBase.h FWCore/Common/interface/LuminosityBlockBase.h
+/**\class LuminosityBlockBase LuminosityBlockBase.h
+ FWCore/Common/interface/LuminosityBlockBase.h
```
Comment (wddgit) [inline FWCore/Common/interface/LuminosityBlockBase.h]: I do not know the answer to this question. What does doxygen do if you split a line that is a doxygen command across multiple lines?

File: FWCore/Common/src/LuminosityBlockBase.cc [hunk 2]
```diff
@@ -78,15 +75,17 @@ namespace edm
             }
 
             std::pair<TriggerNamesMap::iterator, bool> ret =
-               triggerNamesMap.insert(std::pair<edm::ParameterSetID, edm::TriggerNames>(triggerResults.parameterSetID(), triggerNames));
+               triggerNamesMap.insert(std::pair<edm::ParameterSetID,
+   edm::TriggerNames>(triggerResults.parameterSetID(), triggerNames));
```
Comment (wddgit) [inline FWCore/Common/src/LuminosityBlockBase.cc]: Nice that it split the long line, but I would consider the new indentation here to be erroneous.

File: FWCore/Common/src/LuminosityBlockBase.cc [hunk 3]
```diff
@@ -57,8 +52,10 @@ namespace edm
          return &iter->second;
       }
 
-      // Look for the parameter set containing the trigger names in the parameter
-      // set registry using the ID from TriggerResults as the key used to find it.
+      // Look for the parameter set containing the trigger names in the
+   parameter
```
Comment (wddgit) [inline FWCore/Common/src/LuminosityBlockBase.cc]: The way this line was split will not compile.

File: FWCore/Concurrency/src/SerialTaskQueue.cc [hunk 4]
```diff
@@ -23,88 +23,87 @@ using namespace edm;
...
-     } else {
+  auto nextTask = pushAndGetNextTask(iTask);
+  if
+    likely(nullptr != nextTask) {
+      if
+        likely(nextTask == iTask) {
+          // spawn and wait for all requires the task to have its parent set
+          iWait->spawn_and_wait_for_all(*nextTask);
+        }
+      else {
```
Comment (wddgit) [inline FWCore/Concurrency/src/SerialTaskQueue.cc]: Indentation here looks weird to me. The close bracket aligns with likely and
why an indent and new line for likely at all?

**Comment by wddgit** [comment]
I do not know what the options are, but if I could make it do what I wanted I would leave the indentation alone if it is done consistently with 2, 3, or 4 spaces within a block.

I like zero indentation for public, private, and protected lines and my second choice would be a consistent indentation size with code that follows but not 1 space.

I also prefer to indent inside namespaces.

I love getting rid of tabs, fixing errors where within the same block where the indentation is not consistent and adding indentation that missing and needed ...

It also bothers me some that we are making it so much harder to find the history and trace who wrote which line of code ... And there are so many changes here it would be very difficult to actually review it all.

**Comment by davidlange6** [comment]
should be 80 characters
https://google.github.io/styleguide/cppguide.html#Line_Length

**Comment by davidlange6** [comment]
i guess we were expected to have written if ( likely(nullptr != nextTask)) { ? [which seems to be the way its typically done, but I don't quickly find a standard on this]

**Comment by davidlange6** [comment]
we will iterate again...closing for now.

**Pull request closed without merging**